client/app/config/US/common.constants.js

```diff
@@ -1,130 +1,130 @@
+        vpsPage: "https://www.ovh.us/manager/dedicated/index.html#/configuration/vps/{vps}?landingTo=vps",
+        networks: "https://www.ovh.us/manager/dedicated/index.html#/configuration?landingTo=networks",
+        cdnPage: "https://www.ovh.us/manager/dedicated/index.html#/configuration/cdn/{cdn}?landingTo=networks",
+        renew: "https://www.ovh.us/cgi-bin/order/renew.cgi?domainChooser={serviceName}"
+    })
 // CA eq to en_CA
 // QC eq to fr_CA
 // WS eq to es_US
 // AU eq to en_CA
-.constant("URLS", {
-    "support": {
-        US: "http://www.ovh.us/support/"
-    },
-    "support_contact": {
-        US: "https://www.ovh.us/us/support/"
-    },
-    "website_order": {
-        "cloud-resell-eu": {
-            US: (projectName) =>
-                "https://www.ovh.us/order/express/#/new/express/resume?products=~(~(planCode~'project-eu~productId~'cloud~quantity~1~duration~'P1M~configuration~(~(label~'description~values~(~'"+encodeURIComponent(projectName)+")))))",
+    .constant("URLS", {
+        support: {
+            US: "http://www.ovh.us/support/"
         },
-        "dedicated_server": {
-            US: "https://www.ovh.us/dedicated-servers/"
+        support_contact: {
+            US: "https://www.ovh.us/us/support/"
         },
-        "dedicated_cloud": {
-            US: "https://www.ovh.us/dedicated-cloud/"
-        },
-        "load_balancer": {
-            US: "https://www.ovh.us/solutions/ip-load-balancing/"
-        },
-        pcs: {
-            US: "https://www.ovh.us/public-cloud/storage/object-storage/"
-        },
-        pca: {
-            US: "https://www.ovh.us/public-cloud/storage/cloud-archive/"
-        },
-        vrack: {
-            AU: "https://ca.ovh.com/au/order/express/#/new/express/resume?products=~(~(planCode~'vrack~quantity~1~productId~'vrack))",
-            CA: "https://ca.ovh.com/en/order/express/#/new/express/resume?products=~(~(planCode~'vrack~quantity~1~productId~'vrack))",
-            QC: "https://ca.ovh.com/fr/order/express/#/new/express/resume?products=~(~(planCode~'vrack~quantity~1~productId~'vrack))",
-            WE: "https://us.ovh.com/us/order/express/#/new/express/resume?products=~(~(planCode~'vrack~quantity~1~productId~'vrack))",
-            WS: "https://us.ovh.com/es/order/express/#/new/express/resume?products=~(~(planCode~'vrack~quantity~1~productId~'vrack))"
-        },
-        cloud_disk_array: {
-            US: "https://www.ovh.us/cloud/cloud-disk-array/"
-        },
-        veeam: {
-        }
-    },
-    "guides": {
-        "home": {
-            US: "http://docs.ovh.us"
+        website_order: {
+            "cloud-resell-eu": {
+                US: projectName =>
+                    `https://www.ovh.us/order/express/#/express/review?products=~(~(planCode~'project-eu~productId~'cloud~quantity~1~duration~'P1M~configuration~(~(label~'description~values~(~'${encodeURIComponent(projectName)})))))`
```

Comment:
This is the only real change, all others are ESLINT related.
I know I should have put it in 2 different commits... :)